src/Tasks/Microsoft.NET.Build.Extensions.Tasks/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.NETFramework.targets

```diff
@@ -50,7 +50,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       <!-- 1. The .NET Standard 2.0 support need to be injected -->
       <_RunGetDependsOnNETStandard Condition="'$(DependsOnNETStandard)' == '' AND '$(NETStandardInbox)' != 'true'">true</_RunGetDependsOnNETStandard>
       <!-- 2. The target framework is .NET 4.7.1, which needs to special case some shims -->
-      <_RunGetDependsOnNETStandard Condition="'$(_TargetFrameworkVersionWithoutV)' == '4.7.1'">true</_RunGetDependsOnNETStandard>
+      <_RunGetDependsOnNETStandard Condition="'$(DependsOnNETStandard)' == '' AND '$(_TargetFrameworkVersionWithoutV)' == '4.7.1'">true</_RunGetDependsOnNETStandard>
```

Comment:
I wonder if this condition should check against "not true" instead of empty.  Though, I see that above we are doing the check in a similar manner.

Reply:
What we are really checking here is whether or not we have to run the DependsOnNETStandard task, and so what we check is that if we don't know yet if we depend on it or not (if value is empty) then we might want to run the task.

Reply:
if we added a condition to be "not true", then what if we want to force the property to be false? We would evaluate this condition to be true and run the task anyway which will override the value we wanted to force, which is why we should check against empty.